### ISSUE TITLE
unable to plot in jupyter notebook (can't use a string pattern on a bytes-like object)

### ISSUE THREAD

**Issue by j9ac9k** [UFAL-DSG/pyfst#7]
I just recently installed openfst/pyfst (using it for class) on a python 3 system running Linux; however I'm unable to generate any of the plots (I have graphviz installed)

Using the example code:
```
import fst

t = fst.Transducer()

t.add_arc(0, 1, 'a', 'A', 0.5)
t.add_arc(0, 1, 'b', 'B', 1.5)
t.add_arc(1, 2, 'c', 'C', 2.5)

t[2].final = 3.5

t.shortest_path() # 2 -(a:A/0.5)-> 1 -(c:C/2.5)-> 0/3.5 
```

I get the following error:

```

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
/home/ogi/miniconda3/lib/python3.4/site-packages/IPython/core/formatters.py in __call__(self, obj)
    339             method = _safe_get_formatter_method(obj, self.print_method)
    340             if method is not None:
--> 341                 return method()
    342             return None
    343         else:

fst/_fst.pyx in fst._fst._Fst._repr_svg_ (fst/_fst.cpp:5339)()

fst/_fst.pyx in fst._fst.StdVectorFst.draw (fst/_fst.cpp:18374)()

/home/ogi/miniconda3/lib/python3.4/re.py in sub(pattern, repl, string, count, flags)
    177     a callable, it's passed the match object and must return
    178     a replacement string to be used."""
--> 179     return _compile(pattern, flags).sub(repl, string, count)
    180 
    181 def subn(pattern, repl, string, count=0, flags=0):

TypeError: can't use a string pattern on a bytes-like object

Out[11]:
<StdVectorFst with 3 states>
```

Any suggestions?

**Comment by jurcicek** [comment]
This is being fixed in the #8 PR. Will be merged soon.

**Comment by jurcicek** [comment]
The #8 PR was merged and solves the issue.
